File: 412-fizz-buzz/412-fizz-buzz.go
func fizzBuzz(n int) []string {
    str := []string{}
    
    for i := 1; i <= n; i++ {
        if i % 3 == 0 && i % 5 == 0 {
            str = append(str, "FizzBuzz")
        } else if i % 5 == 0 {
            str = append(str, "Buzz")
        } else if i % 3 == 0 {
            str = append(str, "Fizz")
        } else {
            str = append(str, strconv.Itoa(i))
        }
    } 
    return str
}

File: 412-fizz-buzz/412-fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        
        for i in range(1, n + 1):
            if i % 3 == 0 and i % 5 == 0:
                yield "FizzBuzz"
            elif i % 5 == 0:
                yield "Buzz"
            elif i % 3 == 0:
                yield "Fizz"
            else:
                yield str(i)
                